guides/source/routing.md

```diff
@@ -1403,9 +1453,29 @@ $ bin/rails routes -c Comments
 $ bin/rails routes -c Articles::CommentsController
 ```
 
-TIP: You'll find that the output from `bin/rails routes` is much more readable if you widen your terminal window until the output lines don't wrap.
+TIP: The output from `bin/rails routes` is easier to read if you widen your terminal window until the output lines don't wrap.
+
+### Listing Unused Routes
+
+You can scan your application for unused routes with the `--unused` option. For example:
+
+```bash
+$ rails routes --unused
+Found 8 unused routes:
+
+     Prefix Verb   URI Pattern                Controller#Action
+     people GET    /people(.:format)          people#index
+            POST   /people(.:format)          people#create
+ new_person GET    /people/new(.:format)      people#new
+edit_person GET    /people/:id/edit(.:format) people#edit
+     person GET    /people/:id(.:format)      people#show
+            PATCH  /people/:id(.:format)      people#update
+            PUT    /people/:id(.:format)      people#update
+            DELETE /people/:id(.:format)      people#destroy
+```
 
-### Testing Routes
+Testing Routes
+--------------
 
 Routes should be included in your testing strategy (just like the rest of your application). Rails offers three built-in assertions designed to make testing routes simpler:
```

Comment:
Wondering if the "just like the rest of your application" in brackets is necessary?

```suggestion
Routes should be included in your testing strategy. Rails offers three built-in assertions designed to make testing routes simpler:
```

Reply:
Removing that whole line actually.